alvarotorrestx.md

```diff
@@ -0,0 +1,9 @@
+# Changes Made
```

Comment:
I'd like to clean the project tidy if possible. You can provide these bullet points in the pull request description instead and I will add a note about your contribution to the release notes. :)

Reply:
Gotcha and understood, will do! 👌🏽